docs/source/02_Terminology.md

```diff
@@ -131,7 +132,35 @@ Since in practical terms the time is measured in discrete samples,
 the temporal scope includes the start time sample 
 but does not include the end time sample.
 
-## Time-block [*]
+### Time-block [*]
 
 A contiguous portion of the data recording during which some aspect of the experiment 
 is fixed or noted.
+
+## 2.2 Character sets and restrictions
+
+Starting with HED standard schema versions 8.3.0 and above, HED will allow UTF-8 characters in various settings.
+The types of characters referred to in this specification are:
+
+| Name | Description |
+| ---- | ----------- |
+| `ascii` | utf-8 codes 0 to 127 (single byte) |  
+| `non-ascii` | utf-8 codes greater than 128 (multi-byte) |  
+| `printable` | ascii 32 <= code < 127 |  
+| `letters` | ascii characters a-z and A-Z |  
+| `uppercase` | ascii characters A-Z |  
+|`lowercase` | ascii characters a-z |  
+| `text` | `printable` or `non-ascii` |  
```

Comment:
This is inconsistent with letters. Both should be and/or.

Reply:
Corrected.